src/handler/src/commands/github/issue-pr.ts

```diff
@@ -0,0 +1,447 @@
+				? i18next.t('command.issue-pr.execute.action.close', { lng: locale })
+				: resultState === 'DRAFT'
+				? i18next.t('command.issue-pr.execute.action.draft', { lng: locale })
+				: i18next.t('command.issue-pr.execute.action.open', { lng: locale });
+
+			const text = `${comments}${action}`;
+
+			// timestamp
+			const timestampProperty = Timestamps[resultState];
+
+			const e1: Embed = {
+				author: {
+					icon_url: `${issue.author.avatarUrl}?anticache=${Date.now()}`,
+					name: issue.author.login,
+					url: issue.author.url,
+				},
+				title: `#${issue.number} ${issue.title}`,
+				url: issue.url,
+				footer: { text, icon_url },
+				description: verbose ? IssuePRLookup.formatBody(issue.body) : undefined,
+				color: StateColors[resultState],
+				image: verbose && groups ? { url: groups.url } : undefined,
+				timestamp: isPR(issue) ? issue[timestampProperty]! : issue[timestampProperty as TimestampsWithoutMergedKey]!,
+			};
+
+			// install with
+			const installable = resultState in InstallableState;
+			const e2: Embed =
+				isPR(issue) && installable
+					? addField(e1, {
+							name: i18next.t('command.issue-pr.execute.headings.install', { lng: locale }),
+							value: `\`npm i ${issue.headRepository.nameWithOwner}#${
+								issue.headRef?.name ?? i18next.t('command.issue-pr.execute.unknown', { lng: locale }) ?? ''
+							}\``,
+					  })
+					: e1;
+
+			// reviews
+			const reviews = isPR(issue) ? IssuePRLookup.relevantReviews(issue.author.login, issue.reviews.nodes) : [];
+			const reviewBody = reviews
+				.map((r) => {
+					const isDjsMember =
+						owner === 'discordjs' && ['MEMBER', 'OWNER', 'COLLABORATOR'].includes(r.authorAssociation);
+					const reviewBadge = isDjsMember ? Badges.DJS : '';
+					const reviewLink = `[${r.author.login}](${r.url})`;
+					const reviewState = `${IssuePRLookup.cleanDecision(r.state)}`;
+					return `${reviewBadge} ${reviewLink} ${reviewState}`;
+				})
+				.join('\n');
+
+			const reviewTitle = `${i18next.t('command.issue-pr.execute.headings.reviews', { lng: locale })}${
+				isPR(issue) && issue.reviewDecision ? ` (state: ${IssuePRLookup.cleanDecision(issue.reviewDecision)})` : ''
+			}`;
+
+			const e3: Embed = reviews.length ? addField(e2, { name: reviewTitle, value: reviewBody }) : e2;
+
+			// labels
+			const e4: Embed = issue.labels.nodes.length
+				? addField(e3, {
+						name: i18next.t('command.issue-pr.execute.headings.labels', { lng: locale }),
+						value: issue.labels.nodes
+							.map(
+								(l: { name: string; color: string; url: string }) =>
+									`${IssuePRLookup.label(l.color)}[${l.name}](${l.url})`,
+							)
+							.join(' '),
+				  })
+				: e3;
+
+			this.rest.post(`/channels/${message.channel_id}/messages`, {
+				embed: truncateEmbed(e4),
+			});
+		} catch {}
+	}
+
+	private static buildQuery(owner: string, repository: string, issueID: string) {
+		return `
+		{
+			repository(owner: "${owner}", name: "${repository}") {
```

Comment:
escape quote marks/newlines in the `respository`? to prevent a malformed request if a user put in quotemarks

Reply:
opted for validating the name instead of escaping in https://github.com/Naval-Base/yuudachi/pull/767/commits/147f70847a93da6643b19bebd9590ef5e0ed58ab . regex taken from https://stackoverflow.com/a/59082561, approach seems valid enough from a bit of own testing and reading linked moby issue